### DISCUSSION TITLE
Activated carplay in mib

### DISCUSSION THREAD

**Discussion by veicmario** [Mr-MIBonk/M.I.B._More-Incredible-Bash#129] [General]
![20230219_205851](https://user-images.githubusercontent.com/15074331/220166145-43746d9d-4056-49d9-bc88-8219b54e3201.jpg)

I have activated carplay but when i connect phone nothing happen. I get audi smarthphone option in menu and everything.

**Comment by harman-f** [comment]
That is the AIO you installed. 

did you try to activate again with M.I.B —> log please. 

Have you checked if your USB port is comparable?

**Comment by veicmario** [comment]
Hi, i sent you photos, i tried few times but no lick, two different cables and two phones but no luck.

If i heard right, i have no activation codes in red menu for carplay?

**Comment by harman-f** [comment]
No photos received.

**Comment by veicmario** [comment]
![20230220_192939](https://user-images.githubusercontent.com/15074331/220183203-9182df24-8c0d-4dd2-952a-38f6df99cc6e.jpg)
![20230220_192932](https://user-images.githubusercontent.com/15074331/220183223-d0561e7c-e5ee-4355-8513-6e68825bc6cb.jpg)
![IMG_20230220_184615_991](https://user-images.githubusercontent.com/15074331/220183246-d56c5ad8-3933-4935-8916-a97a3a6a2935.jpg)

**Comment by veicmario** [comment]
[MU1425-JTB-07414.03.17116600A6.log](https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/files/10787254/MU1425-JTB-07414.03.17116600A6.log)

**Comment by harman-f** [comment]
And a picture of your USB port.

**Comment by veicmario** [comment]
![Screenshot_20230220_202015_Facebook](https://user-images.githubusercontent.com/15074331/220184677-d44af0e3-37e6-4dcb-bfb9-660a6a83c781.jpg)

**Comment by harman-f** [comment]
No phone symbol. 

Can you put a USB stick with mp3 files in and play them on unit?
